### ISSUE TITLE
Sharing the same X session?

### ISSUE THREAD

**Issue by AndreaCallia** [weichuntsai0217/xserver#1]
Hi all, is it possible to share the same X session with two different VNC clients?

**Comment by weichuntsai0217** [comment]
Hi Andrea,

I don't know if it is possible. However, maybe you can try this:
https://www.linuxquestions.org/questions/linux-newbie-8/sharing-single-vnc-session-between-two-users-887260/

**Comment by AndreaCallia** [comment]
I have managed :) I modified your `x11vnc` command adding a `-shared` option... now I can try to play online with a friend of mine with some dosbox multiplayer games (although I will be the only one having audio, he will have a muted game)

More in detail, this is the `x11vnc` command that I use in my edited version of your `run-xserver.sh` script:

    x11vnc -shared -forever -passwd $VNC_PASSWORD -display $DISPLAY -rfbport 5900 2> /dev/null

**Comment by weichuntsai0217** [comment]
You did a great job! Thank you for your sharing. This is very useful!
By the way, could you share that what game do you play?

**Comment by AndreaCallia** [comment]
Yes of course... I made some tests with the "Kings of the beach" game... would you like me to upload something?

**Comment by AndreaCallia** [comment]
Ah, one thing: every time that I have to stop the `xserver` image, I first have to delete file `/tmp/.X0-lock`. If I forget to do it, then the image will not start anymore, because it will not manage to create a new session, and I will have to delete the image and make a new one, which means that I have to edit again the script to add the `-shared` option to `x11vnc` ... is there an easy way to automatically delete file `/tmp/.X0-lock` when the image is stopped?
